### PR TITLE
Add meta tests for layer/group/batch norm backward

### PR DESCRIPTION
**Pull request by kundaMwiza**
Fixes #ISSUE_NUMBER

### CODE REVIEW

**Review by ezyang — APPROVED** [review]
Imma accept this, but ideally these would be done as OpInfos rather than one off tests in test_meta.py, unless there is a good reason why we can't

**Comment by kundaMwiza** [comment]
@ezyang Yeah, a more general solution is required for testing backward metas

**Comment by ezyang** [comment]
https://github.com/pytorch/pytorch/pull/109211 may be some inspiration
